tests/entities_column_value_simple_tests.py

```diff
@@ -251,3 +251,186 @@ def test_should_set_number_column_value_to__a_valid_string_value():
 
     #Assert
     eq_(column_value.value, float(value))
+
+def test_should_create_a_status_column_value_with_no_api_input_data():
+
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    column_value = en.cv.create_column_value(column_type, id=id, title=title)
+
+    # Act
+    format = column_value.format()
+
+    #Assert
+    eq_(format, {})
+
+def test_should_create_a_status_column_value_with_api_input_data():
+
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    index_value = {
+        'index': 1,
+    }
+    settings = {'labels': {'0': 'Working on it', '1': 'Done', '2': 'Stuck'}}
+    settings_str = json.dumps(settings)
+    value = json.dumps(index_value)
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,value=value,settings_str=settings_str)
+
+    # Act
+    format = column_value.format()
+
+    #Assert
+    eq_(column_value.value,'Done')
+    eq_(format['index'], 1)
+
+def test_should_set_none_to_status_value():
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    column_value = en.cv.create_column_value(column_type, id=id, title=title)
+
+    # Act 
+    column_value.value = None
+
+    #Assert
+    eq_(column_value.value, None)
+
+
+@raises(e.ColumnValueError)
+def test_should_set_invalid_label_value_to_status_value():
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    settings = {'labels': {'0': 'Working on it', '1': 'Done', '2': 'Stuck'}}
+    settings_str = json.dumps(settings)
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,settings_str=settings_str)
+
+    # Act 
+
+    column_value.value = "Not Done"
+
+def test_should_set_a_status_column_value_with_string_index_value():
+
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    index_value = {
+        'index': '1',
+    }
+    settings = {'labels': {'0': 'Working on it', '1': 'Done', '2': 'Stuck'}}
+    settings_str = json.dumps(settings)
+    value = json.dumps(index_value)
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,value=value,settings_str=settings_str)
+
+    # Act
+    format = column_value.format()
```

Comment:
When I say to test setting the value, I am referring to setting the column_value.value directly through the setter.  This test checks the convert part of the equation, but it does not address the actual setting.

I would suggest for these tests, simplify and always create the column_value with no input data.  Then set the data type that you need to test and assert on that.  No need for formatting either.